extensions/youtube/src/main/java/app/revanced/extension/youtube/settings/LicenseActivityHook.java

```diff
@@ -147,7 +150,34 @@ private static void createToolbar(Activity activity, String toolbarTitleResource
         }
         setToolbarLayoutParams(toolbar);
 
+        // Add SearchView only for ReVancedPreferenceFragment
```

Comment:
Comments end with a dot.

```suggestion
        // Add SearchView only for ReVancedPreferenceFragment.
```